src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/PaymentBuilderTable/partials/RecipientField/RecipientField.tsx

```diff
@@ -1,37 +1,16 @@
-import clsx from 'clsx';
 import React, { type FC } from 'react';
 
-import { useMemberContext } from '~context/MemberContext/MemberContext.ts';
 import UserPopover from '~v5/shared/UserPopover/UserPopover.tsx';
 
 import { type RecipientFieldProps } from './types.ts';
 
-const RecipientField: FC<RecipientFieldProps> = ({ address }) => {
-  const { totalMembers, loading } = useMemberContext();
-  const recipientMember = totalMembers.find(
-    (member) => member.contributorAddress === address,
-  );
-
-  return (
-    <>
-      {!loading && (
-        <div className="flex items-center">
-          <UserPopover
-            size={18}
-            walletAddress={
-              recipientMember ? recipientMember.contributorAddress : address
-            }
-            withVerifiedBadge={!!recipientMember?.isVerified}
-            className={clsx('flex items-center sm:hover:text-blue-400', {
-              'pointer-events-none': loading,
-              'text-warning-400': !recipientMember?.isVerified,
-              'text-gray-900': recipientMember?.isVerified,
-            })}
-          />
-        </div>
-      )}
-    </>
-  );
-};
+const RecipientField: FC<RecipientFieldProps> = ({ address, isLoading }) => (
+  <UserPopover
+    size={18}
+    walletAddress={address}
+    withVerifiedBadge
+    isLoading={isLoading}
```

Comment:
This is just a major issue with how the popovers currently work, but this is further complicating their behaviour.
We are planning a refactor of them, but could we please for the moment just display a skeleton if `isLoading` is true, and stop propagating this flag to both `UserInfoPopover` and `UserPopover` components?